### DISCUSSION TITLE
Comment thread on Traces

### DISCUSSION THREAD

**Discussion by guptakvgaurav** [langfuse/langfuse#2729] [Ideas]
### Describe the feature or potential improvement

As a user i would like to have a comment thread section on the Traces. As a developer i am continuously seeking comments from subject matter experts on the generated results. This helps in continuous improvement of the prompts.

**Comment by marcklingen** [comment]
comments across langfuse are already planned, cc @marliessophie

**Comment by marliessophie** [comment]
We've released a first version of comments across objects in the Langfuse UI. Please see our [changelog]( https://langfuse.com/changelog/2024-08-20-comments) for details. As always - we'd love to hear feedback on the feature!

**Comment by darvin** [comment]
how about API for comments @marliessophie  ?
